src/react-app-env.d.ts

```diff
@@ -1,2 +1,12 @@
 // eslint-disable-next-line
 /// <reference types="react-scripts" />
+
+interface Person {
+  name: string,
+  sex: string;
```

Comment:
Create an enum for the `sex` property.

```suggestion
  name: string;
  sex: string;
```